hudi-client/src/test/java/org/apache/hudi/index/hbase/TestHBaseIndex.java

```diff
@@ -383,22 +337,12 @@ public void testsHBasePutAccessParallelismWithNoInserts() {
     assertEquals(0, hbaseNumPuts);
   }
 
-  @Test
-  public void testsHBaseIndexDefaultQPSResourceAllocator() {
```

Comment:
duplicate test case in org.apache.hudi.index.hbase.TestHBaseQPSResourceAllocator